pkg/config/templates/rancherd-22-addons.yaml

```diff
@@ -24,9 +28,176 @@ resources:
       repo: http://harvester-cluster-repo.cattle-system.svc/charts
       version: "0.2.4-rc2"
       chart: harvester-pcidevices-controller
+      {{- if and .Addons .Addons.harvester_pcidevices_controller }}
+      enabled: {{ .Addons.harvester_pcidevices_controller.Enabled }}
+      {{- else }}
       enabled: false
+      {{- end }}
       valuesContent: |
         image:
           tag: v0.2.4-rc2
         fullnameOverride: harvester-pcidevices-controller
-
+  - apiVersion: harvesterhci.io/v1beta1
+    kind: Addon
+    metadata:
+      name: rancher-logging
+      namespace: cattle-logging-system
+    spec:
+      repo: http://harvester-cluster-repo.cattle-system.svc/charts
+      version: "100.1.3+up3.17.7"
+      chart: rancher-logging
+      {{- if and .Addons .Addons.rancher_logging }}
+      enabled: {{ .Addons.rancher_logging.Enabled }}
+      {{- else }}
+      enabled: false
+      {{- end }}
+      valuesContent: |
+        images:
+          fluentbit:
+            tag: 1.9.5 # replace the default 1.9.3, which has bug in systemd log
+          fluentbit_debug:
+            tag: 1.9.5-debug
+        systemdLogPath: /var/log/journal
+        disablePvc: true
+        additionalLoggingSources:
+          rke2:
+            enabled: true
+          kubeAudit:
+            enabled: true
+            auditFilename: "audit.log"
+            pathPrefix: "/var/lib/rancher/rke2/server/logs/"
+            loggingRef: "harvester-kube-audit-log-ref"
+            fluentbit:
+              logTag: kube-audit
+              tolerations:
+              - key: node-role.kubernetes.io/controlplane
+                value: "true"
+                effect: NoSchedule
+              - key: node-role.kubernetes.io/etcd
+                value: "true"
+                effect: NoExecute
+        fluentd:
+          resources:
+            limits:
+              cpu: 1000m
+              memory: 800Mi
+            requests:
+              cpu: 100m
+              memory: 200Mi
+        fluentbit:
+          resources:
+            limits:
+              cpu: 200m
+              memory: 200Mi
+            requests:
+              cpu: 50m
+              memory: 50Mi
+        eventTailer:
+          name: harvester-default
+          namespace: cattle-logging-system
+          controlNamespace: cattle-logging-system
+          workloadOverrides:
+            containers:
+            - image: banzaicloud/eventrouter:v0.1.0
+              name: event-tailer
+              resources:
+                limits:
+                  cpu: 100m
+                  memory: 100Mi
+                requests:
+                  cpu: 20m
+                  memory: 20Mi
+  - apiVersion: harvesterhci.io/v1beta1
+    kind: Addon
+    metadata:
+      name: rancher-monitoring
+      namespace: cattle-monitoring-system
+    spec:
+      repo: http://harvester-cluster-repo.cattle-system.svc/charts
+      version: "100.1.0+up19.0.3"
```

Comment:
ditto, check if a [newer version](https://github.com/rancher/charts/tree/dev-v2.7/charts/rancher-monitoring/102.0.0%2Bup40.1.2) would be preferred.

Reply:
With an initial test, both `cluster metrics` and `vm metrics` are broken in `rancher-monitoring` `100.1.0+up19.0.3`, need further investigation.

![image](https://user-images.githubusercontent.com/31133476/231263802-6373337c-0dd6-43be-8b5c-51c5a35fc9ce.png)

![image](https://user-images.githubusercontent.com/31133476/231263867-178f177b-6cdc-44a0-bcec-f876d90269ae.png)